tests/test_sourmash.py

```diff
@@ -72,6 +77,28 @@ def test_do_sourmash_compute_output_valid_file():
                    for testdata in (testdata1, testdata2, testdata3))
 
 
+def test_do_sourmash_compute_output_stdout_valid():
+    with utils.TempDirectory() as location:
+        testdata1 = utils.get_test_data('short.fa')
+        testdata2 = utils.get_test_data('short2.fa')
+        testdata3 = utils.get_test_data('short3.fa')
+
+        status, out, err = utils.runscript('sourmash',
+                                           ['compute', '-o', '-',
+                                            testdata1,
+                                            testdata2, testdata3],
+                                           in_directory=location)
+
+        # is it valid json?
+        import json
+        fp = StringIO(out)
+        data = json.load(fp)
```

Comment:
if `out` is a string you could use `json.loads(out)` directly. For my education why do we import json only in the test?